repo/incubating/mysql/versions/0/mysql-frameworkversion.yaml

```diff
@@ -0,0 +1,249 @@
+  connectionString: ""
+  framework:
+    name: mysql
+    kind: Framework
+  parameters:
+  - name: BACKUP_FILE
+    description: "Filename to save the backups to"
+    default: "backup.sql"
+    displayName: "BackupFile"
+    trigger: backup
+  - name: PASSWORD
+    default: "password"
+  tasks:
+    deploy:
+      resources:
+      - mysql.yaml
+    init:
+      resources:
+      - init.yaml
+    pv:
+      resources:
+      - backup-pv.yaml
+    backup:
+      resources:
+      - backup.yaml
+    restore:
+      resources:
+      - restore.yaml
+    load-data:
+      resources:
+      - job.yaml
+    clear-data:
+      resources:
+      - job.yaml
+    query:
+      resources:
+      - job.yaml
+  templates:
+    mysql.yaml: |
+      apiVersion: v1
+      kind: Service
+      metadata:
+        name: mysql
+      spec:
+        ports:
+        - port: 3306
+        selector:
+          app: mysql
+        clusterIP: None
+      ---
+      apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: mysql-pv
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+      ---
+      apiVersion: apps/v1
+      kind: Deployment
+      metadata:
+        name: mysql
+      spec:
+        selector:
+          matchLabels:
+            app: mysql
+        strategy:
+          type: Recreate
+        template:
+          metadata:
+            labels:
+              app: mysql
+          spec:
+            containers:
+            - image: mysql:5.7
+              name: mysql
+              env:
+                # Use secret in real usage
+              - name: MYSQL_ROOT_PASSWORD
+                value: {{PASSWORD}}
+              - name: MYSQL_DATABASE
+                value: kudo
+              ports:
+              - containerPort: 3306
+                name: mysql
+              volumeMounts:
+              - name: mysql-persistent-storage
+                mountPath: /var/lib/mysql
+              livenessProbe:
+                exec:
+                  command: ["mysqladmin", "-p{{PASSWORD}}", "ping"]
+                initialDelaySeconds: 30
+                periodSeconds: 10
+                timeoutSeconds: 5
+              readinessProbe:
+                exec:
+                  # Check we can execute queries over TCP (skip-networking is off).
+                  command: ["mysql", "-p{{PASSWORD}}", "-h", "127.0.0.1", "-e", "SELECT 1"]
+                initialDelaySeconds: 5
+                periodSeconds: 2
+                timeoutSeconds: 1
+            volumes:
+            - name: mysql-persistent-storage
+              persistentVolumeClaim:
+                claimName: mysql-pv
+    backup-pv.yaml: |
+      apiVersion: v1
+      kind: PersistentVolumeClaim
+      metadata:
+        name: backup-pv
+      spec:
+        accessModes:
+          - ReadWriteOnce
+        resources:
+          requests:
+            storage: 1Gi
+    init.yaml: |
```

Comment:
Do we need this job at all, or wasn't it more part of a demo but not the core framework?

I think adding a `demo` folder with a readme that walks through it similar to the one we use in Flink or in our documentation page (with video) https://kudo.dev/docs/examples/backups/ should do the trick.